kernel/src/main/scala/cats/kernel/Eq.scala

```diff
@@ -242,6 +247,11 @@ private[kernel] trait PartialOrderInstances0 extends HashInstances {
     cats.kernel.instances.seq.catsKernelStdPartialOrderForSeq[A]
 }
 
+private[kernel] trait PartialOrderInstances1 extends PartialOrderInstances0 {
+  implicit def catsKernelPartialOrderForSortedMap[K: PartialOrder, V: PartialOrder]: PartialOrder[SortedMap[K, V]] =
```

Comment:
I don't think we need `K: PartialOrder` here.

Reply:
Fixed